dts/arm/xilinx/versalnet_r52.dtsi

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * (C) Copyright 2025, Advanced Micro Devices, Inc.
+ *
+ * Mubin Sayyed <mubin.sayyed@amd.com>
+ */
```

Comment:
keep in style with other dts files in zephyr e.g.
```
/*
 * Copyright 2022 NXP
 *
 * SPDX-License-Identifier: Apache-2.0
 */
```
apply to whole PR

Reply:
sure will fix